### PR TITLE
Recommended channel

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

**Comment by norkans7** [comment]
recommended_channel_1

**Comment by norkans7** [comment]
I think the recommendation text can be removed since the claim blurb is dynamic per channel type

File: templates/channels/channel_claim.haml [hunk 1]
```diff
@@ -570,7 +576,10 @@
       -trans "API Channels"
 
     -for ch_type in channel_types.API
-      .claim-row
+      .claim-row{class:"{% if user_org.get_recommended_channel == ch_type.code %}recommended{% endif %}"}
```
Comment (rowanseymour) [inline templates/channels/channel_claim.haml]: Have we checked if things work correctly when a non-phone channel is recommended?

Comment (norkans7) [reply templates/channels/channel_claim.haml]: Yep

Comment (norkans7) [reply templates/channels/channel_claim.haml]: Checked with FB

**Pull request merged**